src/core/public/chrome/ui/header/home_loader.tsx

```diff
@@ -0,0 +1,143 @@
+import { HomeIcon } from './home_icon';
+
+function findClosestAnchor(element: HTMLElement): HTMLAnchorElement | void {
+  let current = element;
+  while (current) {
+    if (current.tagName === 'A') {
+      return current as HTMLAnchorElement;
+    }
+
+    if (!current.parentElement || current.parentElement === document.body) {
+      return undefined;
+    }
+
+    current = current.parentElement;
+  }
+}
+
+function onClick(
+  event: React.MouseEvent<HTMLAnchorElement>,
+  forceNavigation: boolean,
+  navLinks: ChromeNavLink[],
+  navigateToApp: (appId: string) => void
+) {
+  const anchor = findClosestAnchor((event as any).nativeEvent.target);
+  if (!anchor) {
+    return;
+  }
+
+  const navLink = navLinks.find((item) => item.href === anchor.href);
+  if (navLink && navLink.disabled) {
+    event.preventDefault();
+    return;
+  }
+
+  if (event.isDefaultPrevented() || event.altKey || event.metaKey || event.ctrlKey) {
+    return;
+  }
+
+  if (forceNavigation) {
+    const toParsed = Url.parse(anchor.href);
+    const fromParsed = Url.parse(document.location.href);
+    const sameProto = toParsed.protocol === fromParsed.protocol;
+    const sameHost = toParsed.host === fromParsed.host;
+    const samePath = toParsed.path === fromParsed.path;
+
+    if (sameProto && sameHost && samePath) {
+      if (toParsed.hash) {
+        document.location.reload();
+      }
+
+      // event.preventDefault() keeps the browser from seeing the new url as an update
+      // and even setting window.location does not mimic that behavior, so instead
+      // we use stopPropagation() to prevent angular from seeing the click and
+      // starting a digest cycle/attempting to handle it in the router.
+      event.stopPropagation();
+    }
+  } else {
+    navigateToApp('home');
+    event.preventDefault();
+  }
+}
+
+interface Props {
+  href: string;
+  navLinks$: Observable<ChromeNavLink[]>;
+  forceNavigation$: Observable<boolean>;
+  loadingCount$: Observable<number>;
+  navigateToApp: (appId: string) => void;
+  branding: ChromeBranding;
+}
+
+export function HomeLoader({ href, navigateToApp, branding, ...observables }: Props) {
+  const forceNavigation = useObservable(observables.forceNavigation$, false);
+  const navLinks = useObservable(observables.navLinks$, []);
+  const loadingCount = useObservable(observables.loadingCount$, 0);
+  const label = i18n.translate('core.ui.chrome.headerGlobalNav.goHomePageIconAriaLabel', {
+    defaultMessage: 'Go to home page',
+  });
+
+  return (
+    <EuiHeaderSectionItemButton
+      className="header__homeLoaderNavButton"
+      data-test-subj="homeLoader"
+      onClick={(e: React.MouseEvent<HTMLAnchorElement>) =>
+        onClick(e, forceNavigation, navLinks, navigateToApp)
+      }
+      href={href}
+      // TODO: title seems preferable for tooltip, but need figure out best ally approach to avoid duplication for screen readers: https://www.deque.com/blog/text-links-practices-screen-readers/
```

Comment:
Want to link the issue?

Reply:
I'll create an issue, but I don't think this comment actually should remain here - it's already more accessible than other components, and is better tackled comprehensively. Will remove.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1785 to track screen reader improvements.